lib/client.js

```diff
@@ -0,0 +1,98 @@
+"use strict";
+
+const Promise = require("bluebird");
```

Comment:
That's awesome thanks man! Highly appreciated. If you need anything, don't hesitate to ping :)